app/views/guidance/apply-guidance-for-farmers.njk

```diff
@@ -0,0 +1,319 @@
+    </div>
+
+    <div class="govspeak govuk-!-margin-top-6">
+      <h2 id="what-the-review-is" class="govuk-heading-m">What the review is</h2>
+  
+      <p class="govuk-body">You can get funding for a vet, or team chosen by a vet, to visit your farm and carry out a health and welfare review if you keep:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>beef cattle</li>
+        <li>dairy cattle</li>
+        <li>sheep</li>
+        <li>pigs</li>
+      </ul>
+
+      <p class="govuk-body">A health and welfare review will help you:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>reduce endemic diseases and conditions</li>
+        <li>increase animal productivity</li>
+        <li>improve animal welfare</li>
+      </ul>
+
+      <p class="govuk-body">You can choose the vet that you want to do the review. The review is not an inspection or audit and is optional.</p>
+
+      <h2 id="how-much-funding" class="govuk-heading-m">How much funding you’ll get</h2>
+      
+      <p class="govuk-body">You’ll get funding for each livestock type, not for each animal.</p>
+
+      <p class="govuk-body">You’ll get:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>£522 for a beef cattle review</li>
+        <li>£372 for a dairy cattle review</li>
+        <li>£436 for a sheep review</li>
+        <li>£684 for a pig review</li>
+      </ul>
+
+      <p class="govuk-body">The funding amount for each type of livestock is different because:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>testing may take longer for some types of livestock</li>
+        <li>some results cost more to analyse</li>
+      </ul>
+
+      <p class="govuk-body">The funding will cover the endemic disease or condition testing costs. It will also contribute towards the costs of both your time and the vet’s time.</p>
+
+      <h2 id="what-happens-during-a-review" class="govuk-heading-m">What happens during a review</h2>
+
+      <p class="govuk-body">There’s no set amount of time for a review, but it’s likely to take between 2 and 3 hours of your time and the vet’s time. You may ask the vet to spread it over several regular visits to the farm or do it in a single visit.</p>
+
+      <h3 id="what-the-vet-must-do" class="govuk-heading-s">What you must ask the vet to do</h3>
+
+      <ul class="govuk-list govuk-list--number">
+        <li>Visit your farm and test for the endemic diseases or condition of the livestock type you’re having reviewed to recommended standards - bovine viral diarrhoea (BVD) in dairy and beef cattle, effectiveness of worming treatments in sheep, porcine reproductive and respiratory syndrome (PRRS) in pigs.</li>
```

Comment:
The style of the numbers within the list do not match the prototype which are white numbers in black circles